apps/src/templates/teacherNavigation/UnitCalendar.tsx

```diff
@@ -122,30 +151,47 @@ const UnitCalendar: React.FC = () => {
     state => state.teacherSections.needsReload
   );
 
-  if (isLoading || needsReload) {
+  if (
```

Comment:
Nit / question: This is getting a bit beefy.  Do we think a cleaner code option would be to create a variable like `showLoadingState` for this logic?

That's my instinct, but curious on your thoughts.